Makefile

```diff
@@ -142,19 +142,29 @@ help: ## Display this help
 ## --------------------------------------
 
 .PHONY: test-nocover
+test-nocover: | $(GINKGO)
 test-nocover: ## Run Tests (without code coverage)
-	hack/test-unit.sh
+	ENABLE_UNIT_TESTS=true \
+	ENABLE_INTEGRATION_TESTS=false \
+	hack/test.sh
 
 .PHONY: test
-test: $(GOCOVMERGE)
+test: | $(GINKGO)
 test: ## Run tests
 	@rm -f $(COVERAGE_FILE)
-	hack/test-unit.sh $(COVERAGE_FILE)
+	ENABLE_UNIT_TESTS=true \
+	ENABLE_INTEGRATION_TESTS=false \
+	hack/test.sh $(COVERAGE_FILE)
 
 .PHONY: test-integration
-test-integration: $(ETCD) $(KUBE_APISERVER)
+test-integration: | $(GINKGO) $(ETCD) $(KUBE_APISERVER)
 test-integration: ## Run integration tests
-	KUBECONFIG=$(KUBECONFIG) hack/test-integration.sh $(INT_COV_FILE)
+	@rm -f $(INT_COV_FILE)
+	ENABLE_UNIT_TESTS=false \
+	ENABLE_INTEGRATION_TESTS=true \
+	KUBECONFIG=$(KUBECONFIG) \
```

Comment:
I think the `KUBECONFIG` here is a left over from long ago when these tests used to use kind. It shouldn't be needed, and the tests pass locally when I removed it if you want to remove it in this MR.